### ISSUE TITLE
Accessibility on iOS: hamburger and menu buttons not responsive

### ISSUE THREAD

**Issue by ThomDietrich** [home-assistant/frontend#8643]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

My brother is blind and uses VoiceOver to use his iPhone. See this short intro: https://www.youtube.com/watch?v=qDm7GiKra28

Inside the Home Assistant iOS app he is not able to open the hamburger menu, nor the menu on the right side.

- The hamburger menu doesn't open
  * Clicking the hamburger menu doesn't do anything
  * The blind user can't switch to another dashboard, nor settings or any other of the valuable views
  * Even with the sidebar opened up (done by temporary disabling VoiceOver) no element in the sidebar can be selected

- Menu to the top right doesn't open
  * same issue as for the hamburger menu

This issue is also tracked in https://github.com/home-assistant/iOS/issues/1538

### Describe the behavior you expected

The VoiceOver user should be able to use the app normally.

### Steps to reproduce the issue

1. Activate VoiceOver (iPhone - I did not test on Mac)
2. Open the Home Assistant app
3. Experience very limited functionality

### What version of Home Assistant Core has the issue?

core-2021.3.2

**Comment by ThomDietrich** [comment]
Duplicate of https://github.com/home-assistant/frontend/issues/8178
